### PR TITLE
Patch2

### PR DESCRIPTION
**Pull request by AnshitaVishwa**
Fixes # 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

Resolved the issue #782 

#### Languages Used:
- Ruby

#### Files Added:
- Shaker_sort.rb

### CODE REVIEW

**Comment by AnshitaVishwa** [comment]
sir i have been assigned the issue #782 . in this PR i have implemented the shaker sort algorithm in ruby language.

**Comment by tsah20** [comment]
Please fill your checklist completely and mention the algorithm in your PR

File: shaker_sort/Shaker_sort.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+def shaker_sort(input, length)
```
Comment (jainaman224) [inline shaker_sort/Shaker_sort.rb]: Fix folder name. Change it to `Shaker_Sort`.

Comment (AnshitaVishwa) [reply shaker_sort/Shaker_sort.rb]: sir when i changed the folder name and the  i again commited my changes , 5 more files got renamed and added in this. Now i m stuck sir. Have i done any mistake?

File: Shaker_Sort/Shaker_sort.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+def shaker_sort(input, length)
```
Comment (jainaman224) [inline Shaker_Sort/Shaker_sort.rb]: File name should be `Shaker_Sort`. See the second `s`.

File: Shaker_Sort/Shaker_sort.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  issort=true
+  while issort
```
Comment (jainaman224) [inline Shaker_Sort/Shaker_sort.rb]: Please add some comments.

File: Shaker_Sort/Shaker_sort.rb [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    issort=false
+    i=0
```
Comment (jainaman224) [inline Shaker_Sort/Shaker_sort.rb]: Space around operators like `=`, `+` improves code quality.

**Pull request closed without merging**